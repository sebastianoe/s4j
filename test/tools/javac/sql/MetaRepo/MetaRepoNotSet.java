/*
 * @test  /nodynamiccopyright/
 * @summary Throw error when meta repo is not set but needed
 * @compile/fail/ref=MetaRepoNotSet.out MetaRepoNotSet.java
 */
// key: compiler.err.sql.metarepo.not.available
public class MetaRepoNotSet {
    public static void main(String[] args) {
    	Integer i1 = SQL[SELECT 42 FROM line_item];
    }
}

