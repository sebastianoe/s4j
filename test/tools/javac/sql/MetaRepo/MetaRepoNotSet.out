error: The meta repository option needs to be set when SQL queries should be processed
1 error